packages/grant-explorer/src/features/contributors/DonationsTable.tsx

```diff
@@ -83,32 +85,34 @@ export function DonationsTable(props: {
                           <div className="flex items-center">
                             <img
                               className="w-4 h-4 mr-2"
-                              src={CHAINS[contribution.chainId]?.logo}
+                              src={
+                                CHAINS[contribution.chainId as ChainId]?.logo
+                              }
                               alt="Round Chain Logo"
                             />
                             <Link
                               className={`underline inline-block lg:pr-2 lg:max-w-[200px] max-w-[75px] 2xl:max-w-fit truncate`}
-                              title={contribution.roundName}
+                              title={contribution.round.roundMetadata.name}
                               to={`/round/${
                                 contribution.chainId
                               }/${contribution.roundId.toLowerCase()}`}
                               target="_blank"
                             >
-                              {contribution.roundName}
+                              {contribution.round.roundMetadata.name}
                             </Link>
                             <ChevronRightIcon className="h-4 inline lg:mx-2" />
                           </div>
                           <Link
                             className={`underline inline-block lg:pr-2 lg:max-w-[300px] max-w-[75px] 2xl:max-w-fit truncate`}
-                            title={contribution.projectTitle}
-                            to={`/round/${
-                              contribution.chainId
-                            }/${contribution.roundId.toLowerCase()}/${contribution.roundId.toLowerCase()}-${
-                              contribution.applicationId
-                            }`}
+                            title={contribution.application.project.name}
+                            to={`/round/${contribution.chainId}/${
+                              getConfig().allo.version === "allo-v1"
+                                ? `${contribution.roundId.toLowerCase()}/`
```

Comment:
not sure if there's a need to check if it's allo v1 or v2 here

 you can just do `contribution.roundId.toString().toLowerCase()`

plus the config could be set to allo v1 and this could be an allo v2 round and viceversa